pkg/types/vsphere/validation/platform.go

```diff
@@ -104,3 +110,29 @@ func validateFolder(p *vsphere.Platform, fldPath *field.Path) field.ErrorList {
 
 	return allErrs
 }
+
+func validateNetwork(p *vsphere.Platform, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	vim25Client, _, err := vspheretypes.CreateVSphereClients(context.TODO(),
```

Comment:
The validation in `pkg/types` should only do static checking: It should not connect to the vCenter. For that type of checking, add validation to https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/vsphere/validation.go.